apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/User/GetUserProfileResponseQuery.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
+
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.User;
+
+public static partial class GetUserProfileResponseQuery
+{
+    [Query]
+    public static async Task<GetUserProfileResponse> GetUserProfileAsync(
+        [Service] IMarketParticipantClient_V1 client) =>
+        await client.UserUserprofileGetAsync();
+}
```

Comment:
I know this is just mirroring the backend types, but I feel like the naming goes against our conventions. I would change this class to `UserProfileNode` and then include a descriptor that also changes the name of the `GetUserProfileResponse` type. Like this:

```cs
descriptor.Name("UserProfile");
```

Reply:
Agree :-)